src/server/protocol.ts

```diff
@@ -2956,6 +2956,12 @@ namespace ts.server.protocol {
         Smart = "Smart",
     }
 
+    export enum SemicolonPreference {
+        Ignore = "ignore",

```

Comment:
I assume we have to use ignore for this repo, but what does the diff look like if you choose insert?

Reply:
Generally the same. I sampled some changes after auto-inserting semicolons, and I think the only pattern I saw was in single-line object types, `type X = { x: string };` becomes `type X = { x: string; };`